.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js: 
+  - "node"
+install:
+  - npm install -g mocha
```

Comment:
yes right, we don't need global installation.
But the installation doesn't affect our project. 
It will be installed on travis system while running test.